components/app/project-detail/ProjectSingleDaoProposal.vue

```diff
@@ -64,11 +64,13 @@
 
         <!-- Action (lg only) -->
         <div class="hidden lg:block lg:col-span-2">
+          <app-link to="https://vote.oceanprotocol.com/#/officialoceandao.eth">
```

Comment:
chore(blocking): this link also needs to be applied to the other action inside ProjectSingleDaoProposal. See the bottom of the component to find the other part.